program/Talk - Nicholas Shook - Clojure, AWS & Lambda/README.md

```diff
@@ -1,30 +1,26 @@
-# "Your Session Title" - \<Your Name\>
+# "Serverless Clojure on AWS" - Nicholas Shook
 
 > This template should be copied to the folder
 > `program/<Name> - <Title>/README.md`.
 >
 > Remove this notice and replace any TBA sections you can. Feel free to leave
 > those you cannot.
 
-**Format**: *TBA - e.g. "Talk, 40 minutes" or "Workshop, 2 hours"*
+**Format**: Workshop, 2 hours
```

Comment:
yup, that's what I thought too, but then I thought you wanted me to go in-depth on a workshop. I'll prepare for just a 40 minute presentation.